ingest/mappings/dataset.json

```diff
@@ -0,0 +1,208 @@
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "dcoCommunities": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "portalGroups": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "projects": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "citations": {
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "uri": {
+            "type": "string"
+          }
+        }
+      },
+      "distributions": {
+        "properties": {
+          "accessURL": {
+            "type": "string"
+          },
+          "files": {
+            "properties": {
+              "downloadURL": {
+                "type": "string"
+              },
+              "name": {
+                "type": "string"
+              },
+              "uri": {
+                "type": "string"
+              }
+            }
+          },
+          "name": {
+            "type": "string"
+          },
+          "uri": {
+            "type": "string"
+          }
+        }
+      },
+      "dcoId": {
+        "type": "string"
+      },
+      "doi": {
+        "type": "string"
+      },
+      "isDcoPublication": {
+        "type": "boolean"
+      },
+      "mostSpecificType": {
+        "type": "string"
+      },
+      "publicationYear": {
+        "type": "string"
+      },
+      "title": {
```

Comment:
Just did. And working.

Issue: at 
search_sortby: [
                    {'display':'Year','field':'publicationYear'},
                    {'display':'Title','field':'title.exact'}
                ],
When we choose to sort by Title, it is only showing it in Descending order. I have tried things like "order":"asc"/"count"/"term"/"reverse_term"/..... but still, when we choose to order by title, it is descending order.